### ISSUE TITLE
Error from statusMatrix

### ISSUE THREAD

**Issue by jcubic** [isomorphic-git/isomorphic-git#915]
I use Browser and unpkg.com, and statusMatrix throw errors without any explanations:
I've tried:

```javascript
var matrix = await git.statusMatrix({dir});
var matrix = await git.statusMatrix({dir, pattern: './*'});
var matrix = await git.statusMatrix({dir, pattern: '.'});
```
it throws:

1. `Uncaught (in promise) TypeError: Cannot read property 'lastIndexOf' of undefined TypeError: Cannot read property 'lastIndexOf' of undefined`

2. `Cannot read property 'replace' of undefined`

**Comment by wmhilton** [comment]
See if this was fixed by v0.70.4 #923. It sounds like something that might have fixed.

**Comment by jcubic** [comment]
it's working again. thanks.
